<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="4" total="6" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-07-17T21:12:28 IST" name="HubSpot Application Test Automation Suite" finished-at="2021-07-17T21:12:33 IST" duration-ms="5468">
    <groups>
    </groups>
    <test started-at="2021-07-17T21:12:28 IST" name="HubSpot Application Chrome Test Cases" finished-at="2021-07-17T21:12:30 IST" duration-ms="2231">
      <class name="com.testcases.LoginPageTest">
        <test-method is-config="true" signature="setUpTest(java.lang.String)[pri:0, instance:com.testcases.LoginPageTest@1a72a540]" started-at="2021-07-17T21:12:28 IST" name="setUpTest" finished-at="2021-07-17T21:12:30 IST" duration-ms="1989" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method signature="verifyLoginPageTitleTest()[pri:1, instance:com.testcases.LoginPageTest@1a72a540]" started-at="2021-07-17T21:12:30 IST" name="verifyLoginPageTitleTest" finished-at="2021-07-17T21:12:30 IST" duration-ms="15" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [HubSpot Login] but found [Checking browser]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [HubSpot Login] but found [Checking browser]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.testcases.LoginPageTest.verifyLoginPageTitleTest(LoginPageTest.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginPageTitleTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.testcases.LoginPageTest@1a72a540]" started-at="2021-07-17T21:12:30 IST" name="tearDown" finished-at="2021-07-17T21:12:30 IST" duration-ms="82" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.testcases.LoginPageTest -->
    </test> <!-- HubSpot Application Chrome Test Cases -->
    <test started-at="2021-07-17T21:12:30 IST" name="HubSpot Application FireFox Test Cases" finished-at="2021-07-17T21:12:33 IST" duration-ms="3222">
      <class name="com.testcases.LoginPageTest">
        <test-method is-config="true" signature="setUpTest(java.lang.String)[pri:0, instance:com.testcases.LoginPageTest@1115ec15]" started-at="2021-07-17T21:12:30 IST" name="setUpTest" finished-at="2021-07-17T21:12:33 IST" duration-ms="2746" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[FF]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method signature="verifyLoginPageTitleTest()[pri:1, instance:com.testcases.LoginPageTest@1115ec15]" started-at="2021-07-17T21:12:33 IST" name="verifyLoginPageTitleTest" finished-at="2021-07-17T21:12:33 IST" duration-ms="5" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [HubSpot Login] but found [Checking browser]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [HubSpot Login] but found [Checking browser]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.testcases.LoginPageTest.verifyLoginPageTitleTest(LoginPageTest.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginPageTitleTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.testcases.LoginPageTest@1115ec15]" started-at="2021-07-17T21:12:33 IST" name="tearDown" finished-at="2021-07-17T21:12:33 IST" duration-ms="463" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.testcases.LoginPageTest -->
    </test> <!-- HubSpot Application FireFox Test Cases -->
  </suite> <!-- HubSpot Application Test Automation Suite -->
</testng-results>
